coronavirus.md

```diff
@@ -69,12 +69,31 @@ Coming soon ...
 
 ## Models
 There are various types of models in epidemiology, which are used for prediction of prevalence (total number of infected) or the duration of an epidemic spreading processes. One of the most commonly used types is compartmental models, where population is divided into compartments of people with the same properties, e.g. people who are susceptible to virus, who are recovered etc.
-There are three main types of compartmental models in epidemiology: 
+There are three main types of compartmental models in epidemiology based on which many other models can be built upon: 
     Susceptible-Infected (SI), where each person can be just in two states, either susceptible (S) or infected (I); 
      Susceptible-Infected-Recovered (SIS), where each person can be in one of two states, susceptible (S) or infected (I), but then can become susceptible again;
      Susceptible-Infected-Recovered (SIR), where each person can be  in one of three states, susceptible (S), infected (I) or recovered (R).
      
+
+In **SIR model** one can study the number of people in each of the compartments, denoted by variable S, I and R correspondingly. SIR system can be  expressed by the set of ordinary differential equations proposed by [O. Kermack and Anderson Gray McKendrick](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology). 
+The simulations of SIR model are shown in the python script [here](https://github.com/Liyubov/heterogeneous-dynamics-on-networks/blob/master/code_network_heterogen_models/spreading_SIR.py).
+
+The main take-away message from the SIR model simulations is that we observe so-called peak of epidemic, when the number of infected people reaches its maximum. This moment of when size of epidemic outbreak is maximal depends on SIR model parameters: probability of epidemic transmission and recovery from the disease. 
+
+Another model, which has been proven to be much more realistic for some epidemics spreading models is **SEIR model** [1]. 
+The main idea of this model is that for many important infections there is a significant incubation period during which individuals have been infected but are not yet infectious themselves. During this period the individual is in a special state E (exposed), additional state to three states of parts of population in SIR model. 
```

Comment:
You need double space between paragraphs - see http://playbook.datopian.com/style-guide/markdown

Reply:
ok, tnx!